### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by GaryLeeONE** [noriolg/ORIE4741_project#14]
**Summary**
This project is about predicting the Airbnb listing prices in NYC before and during the COVID-19 pandemic. The team wanted to compare two different models trained on different datasets to see how COVID has influenced the Airbnb listings.

**Things I liked**
* The report included a very well-structured table that clearly shows each step of the project: data cleaning/manipulation, then EDA, and finally model fitting and selection, which is a good workflow for a data analysis project like this one.
* The team included several different types of plots in the EDA section, such as violin plots, bar charts, histograms, etc., which helped me get a better understanding of the datasets.
* The team discussed the WMD and Fairness aspects of the models relatively well. They have conducted a good amount of research on the topic and positioned the models and their purposes appropriately.

**Things that could be improved**
* The report is missing important details in the model fitting process, specifically how the team arrived at the parameter values for the models that require parameter tuning such as random forest and XGBoost. (The parameter tuning process should be done via cross validation.)
* There are some parts of the report that include values and concepts that are incorrect. For example, the team wrote that the more trees in the RF model, the more unlikely it is to overfit, which is apparently not true. Also, in the model training and selection part, there are negative R-squared values for the proxgrad method which should be impossible based on the definition of the R-squared metric. This indicates that the team has misunderstood some concepts or has implementation problems in the code, making the results much less convincing.
* I believe the team missed an opportunity to use the **pre-COVID** model to predict the prices for the **post-COVID** dataset, and conversely, use the  **post-COVID** model to predict the prices for the **pre-COVID** dataset. This way, we can very convincingly see how the difference of COVID has an effect on the listing prices. For example, for the **exact same listing**, I should expect the pre-COVID model to predict a higher value compared to the post-COVID model in general. This would create a true "parallel" comparison.
